src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/components/MethodolodyStatusForm.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  BasicMethodology,
+  MethodologyPublishingStrategy,
+  MethodologyStatus,
+} from '@admin/services/methodologyService';
+import { Release } from '@admin/services/releaseService';
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import ButtonText from '@common/components/ButtonText';
+import { Form, FormFieldRadioGroup } from '@common/components/form';
+import FormFieldTextArea from '@common/components/form/FormFieldTextArea';
+import FormFieldSelect from '@common/components/form/FormFieldSelect';
+import Yup from '@common/validation/yup';
+import { Formik, FormikHelpers } from 'formik';
+import React from 'react';
+import { StringSchema } from 'yup';
+
+export interface FormValues {
+  status: MethodologyStatus;
+  latestInternalReleaseNote: string;
+  publishingStrategy?: MethodologyPublishingStrategy;
+  withReleaseId?: string;
+}
+
+// EES-2163 Replace with real list when EES-2264 done.
+const fakeUnpublishedReleases: Release[] = [
+  {
+    id: 'rel-1',
+    title: 'Release 1',
+  } as Release,
+  {
+    id: 'rel-2',
+    title: 'Release 2',
+  } as Release,
+];
+
+interface Props {
+  isPublished?: string;
+  methodologySummary: BasicMethodology;
+  showWithRelease?: boolean; // EES-2163 - flag for showing publishing strategy sections for testing, remove when BE done.
+  unPublishedReleases?: Release[];
+  onCancel: () => void;
+  onSubmit: (values: FormValues, actions: FormikHelpers<FormValues>) => void;
```

Comment:
Ah! Sorry, I just twigged what we've done here.

Think it'd be a good idea to move the `useFormSubmit` (from the parent page) into here. We want to keep that hook as close as possible, because we sometimes need to map errors from the server response to the form fields. It's a good idea to keep these things co-located.

Would suggest that we also change this prop to the following:

```ts
handleSubmit: (values: FormValues) => void;
```

This decouples the parent from the underlying Formik implementation details.

This is pretty much the same pattern as `ReleaseStatusForm` :smile: